telemetry/definitions/commonDefinitions.json

```diff
@@ -850,6 +887,29 @@
             "name": "apprunner_viewServiceLogs",
             "description": "View the App Runner service logs (the logs produced by App Runner)"
         },
+        {
+            "name": "auth_addConnection",
+            "description": "Called when user clicks on add New Connections from any of entry points. fields captures snapshot of user state upon reaching a result",
+            "metadata": [
+                { "type": "source"},
+                { "type": "authUIElement", "required": false},
+                { "type": "unlockedTabs", "required": false},
+                { "type": "authConnectionsCount", "required": false},
+                { "type": "credentialSourceId", "required": false},
+                { "type": "invalidInputFields", "required": false},
+                { "type": "region", "required": false },
+                { "type": "reason", "required": false},
+                { "type": "result"}
+            ]
+        },
+        {
+            "name": "auth_switchRoles",
+            "description": "Called when user clicks yes or no to switch role upon adding new connection",
```

Comment:
We should make the name and description more specific because this is for a very specific UX. Not everything needs a formal definition either. It'd be ok to re-use `ui_click` and attach `authConnectionsCount` to the event instead.